### DISCUSSION TITLE
Any reason for not having Relationships.oneToMany?

### DISCUSSION THREAD

**Discussion by borsemayur2** [eveningkid/denodb#220] [Q&A] [answered]
As per the doc, we have to use `belongsTo` with one-to-many but we can use `Relationhips.oneToOne` that does the mapping automatically. Any reason for this or it's not implemented yet?

**Comment by eveningkid** [comment, marked as answer]
That's true, `oneToOne` existed because it was simpler than calling `belongsTo` twice.

There was however an inconsistency, you're right. But it's now fixed!

As `belongsTo` is now used differently (see [one-to-many](https://eveningkid.com/denodb-docs/docs/guides/one-to-many#add-a-foreign-key)), this should solve the problem.
